.github/workflows/issue.yml

```diff
@@ -0,0 +1,29 @@
+name: Labeling new issue
```

Comment:
Why do we need to contain this file in the PR?

Reply:
sorry, wrongly added. will remove this file. but feel free to review rest.